openrpc.json

```
@@ -0,0 +1,1454 @@
+{
+  "openrpc": "1.0.0-rc1",
+  "info": {
+    "version": "0.0.1",
+    "title": "Digital Asset API"
+  },
+  "servers": [
+    {
+      "url": "https://rpc.metaplex.com/this_is_fake"
```

```suggestion
      "url": "https://aura-mainnet.metaplex.com/"
```
We also have https://aura-devnet.metaplex.com/